app/assets/stylesheets/schools/dashboard.scss

```diff
@@ -7,6 +7,23 @@
     .govuk-summary-list__actions {
       width: 13%;
     }
+    .govuk-summary-list__missing {
+      border-left: 10px solid #1d70b8;
+      padding-left: 15px;
+    }
+    .govuk-summary-list__missing-heading {
+      margin-bottom: 10px;
+    }
+  }
+
+  .app-filter {
+    border: govuk-colour("light-grey") 4px solid;
+    padding: 20px;
+    margin-bottom: 20px;
+
+    .govuk-radios__label {
+      padding-right: 5px;
+    }
   }
```

Comment:
This is super picky, but could this CSS get moved to something like `assets/stylesheets/filter.scss` (or `assets/stylesheets/schools/filter.scss`), even though it's only a few lines?  That might help us find it again in future, and avoid any issues if we add any other filters.

Reply:
done

Reply:
Thank you!